modules/aws-velero/README.md

```diff
@@ -30,8 +57,21 @@ module "velero" {
 }
 ```
 
-## Links
+To use IAM Roles support for Service Account:
```

Comment:
Minor nit: the "commercial" name of this feature is just "IAM Roles for Service Accounts" (IRSA).

Reply:
fixed